### ISSUE TITLE
Structure factor calculations

### ISSUE THREAD

**Issue by biochem-fan** [project-gemmi/gemmi#180]
I have three comments on `gemmi sfcalc`.

1. I would like to have an option to get "normalized" values such that F(0,0,0) equals the total number of electrons. In other words, could you put an option to call `gemmi.IT92_normalize()`?

2. When I run `gemmi sfcalc --to-mtz=output.mtz input.cif`, the program complains `ERROR: Option --to-mtz=output.mtz works only with macromolecular structures`. Why is this explicitly blocked?

3. With `--for=mott-bethe`, F(0,0,0) becomes NaN. I guess this results from (Z - f_xray) / s^2 = 0 / 0 but physically this should be a finite number. Keitaro pointed out that this can be calculated as sum (a_i * b_i) / (32 * r_Bohr * PI ^ 2) where a_i and b_i are n-Gaussian representation of X-ray scattering factors. You can derive this by applying l'Hopital's rule twice to the Mott-Bethe formula. Could you please implement it? Otherwise, `sfcalc` should not write F(0,0,0) because NaN is not the right answer and causes problems in downstream applications.

Meanwhile, my workaround is:

```
gemmi sfcalc --dmin=0.8 --for=mott-bethe input.cif | sed -e s,\(,,g -e s,\),,g|grep -v nan| awk '{printf "% 4d% 4d% 4d%8.2f%8.2f\n", $1, $2, $3, $4, 0.1}' > output.hklf3
# then convert this SHELX HKLF3 style file to MTZ or whatever using iotbx.reflection_file_converter
```

Thanks in advance for your consideration.

**Comment by wojdyr** [comment]
1. I added option `--normalize-it92`.

2. was blocked because it wasn't implemented. Initially, sfcalc worked only with macromolecular files. Then it was extended to work with (small molecule) CIF files, first for calculating individual reflections only. Then support for small mol. with --dmin was added. Now I added writing mtz files.
It's not ideal that the program works differently depending if the input is mmCIF or CIF, but it's also not a big problem.

3. I just removed F000. It was only outputted for CIF input, not for mmCIF. (I you'd need it, let me know).
    I probably should also remove systematically absent reflections.
